+sw_tests/+unit_tests/unittest_spinw_field.m

```diff
@@ -0,0 +1,59 @@
+classdef unittest_spinw_field < sw_tests.unit_tests.unittest_super
+    methods (Test)
+        function test_incorrect_shape_field_raises_error(testCase)
+            swobj = spinw();
+            testCase.verifyError(...
+                @() field(swobj, [0; 0]), ...
```

Comment:
Is it worth testing this for a vector of length 1 and 4 as well with an input argument like `nelem` or similar?